### PR TITLE
VITA: add support for booting game directly via bubble

### PR DESCRIPTION
**Pull request by cy33hc**
Add support for booting a pre configured game directly via a custom bubble similar to RetroArch on the PSVita.

### CODE REVIEW

**Comment by rsn8887** [comment]
It is not acceotable as it is. You can’t add executables to the code base.

**Comment by cy33hc** [comment]
@rsn8887 Reverted the changes for launcher code. Please review again. Thx.

File: backends/platform/sdl/psp2/psp2-main.cpp [hunk 1]
```diff
@@ -52,9 +54,35 @@ int main(int argc, char *argv[]) {
...
+	if (strstr(boot_params,"psgm:play"))
+	{
+		char *param1 = strstr(boot_params, "&param=");
+		char *param2 = strstr(boot_params, "&param2=");
+		if (param1 != NULL && param2 != NULL)
+		{
+			char path[256];
+			char game_id[64];
+			param1 += 7;
+			memcpy(path, param1, param2 - param1);
```
Comment (rsn8887) [inline backends/platform/sdl/psp2/psp2-main.cpp:68]: Don't subtract pointers like this, please find a less hacky way.

File: backends/platform/sdl/psp2/psp2-main.cpp [hunk 2]
```diff
@@ -52,9 +54,35 @@ int main(int argc, char *argv[]) {
...
+			path[param2-param1] = 0;
```
Comment (rsn8887) [inline backends/platform/sdl/psp2/psp2-main.cpp:69]: Please don't subtract pointers like this, use a less hacky way. Thank you.

**Review by rsn8887 — COMMENTED** [review]
Looks almost good to merge, but please don't subtract distinct pointer variables.

**Comment by rsn8887** [comment]
I reviewed it, please update, and if it is ok, I will merge.

**Comment by cy33hc** [comment]
@rsn8887 good point. I'll have that fixed.

**Comment by rsn8887** [comment]
Awesome, thanks! One last request, can you squash commits so there's only one commit called `VITA: add support for booting via bubble`

**Comment by cy33hc** [comment]
@rsn8887 I messed up the squash. I'll abandon the PR and create again.

**Pull request closed without merging**